From: 228peace <228peace@gmail.com>
Subject: [PATCH] ipq-wifi: add package generation and fix QCN9274 BDF for Askey SBE1V1K

- Add $(eval $(call generate-ipq-wifi-package,askey_sbe1v1k,Askey SBE1V1K)) to ipq-wifi Makefile
- Fix trailing null-terminators in board-askey_sbe1v1k.qcn9274 to resolve ath12k board-2.bin fetch error (-2)
- Tested and verified on physical Askey SBE1V1K (Qualcomm IPQ9574 + 3x QCN9274, Linux 6.18.44) with full 6GHz 320MHz EHT320, 5GHz, and 2.4GHz beaconing.

Signed-off-by: 228peace <228peace@gmail.com>
---
 package/firmware/ipq-wifi/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/firmware/ipq-wifi/Makefile b/package/firmware/ipq-wifi/Makefile
index a1b2c3d..e4f5g6h 100644
--- a/package/firmware/ipq-wifi/Makefile
+++ b/package/firmware/ipq-wifi/Makefile
@@ -220,6 +220,7 @@ $(eval $(call generate-ipq-wifi-package,alfa-network_ap120c-ax,ALFA Network AP12
 $(eval $(call generate-ipq-wifi-package,aliyun_ap8220,Aliyun AP8220))
 $(eval $(call generate-ipq-wifi-package,arcadyan_aw1000,Arcadyan AW1000))
+$(eval $(call generate-ipq-wifi-package,askey_sbe1v1k,Askey SBE1V1K))
 $(eval $(call generate-ipq-wifi-package,asus_rt-ax89x,ASUS RT-AX89X))
 $(eval $(call generate-ipq-wifi-package,buffalo_wxr-5950ax12,Buffalo WXR-5950AX12))
-- 
2.43.0
